tests/python/relay/test_vm.py

```diff
@@ -955,5 +956,17 @@ def test_get_input_index():
     assert vm_factory.get_input_index("invalid") == -1
 
 
+@tvm.testing.requires_llvm
+def test_benchmark():
```

Comment:
See comment above on using unittest.patch() so you can test the benchmark() function directly.